lib/constants.js

```diff
@@ -1,16 +1,23 @@
+/* eslint-disable no-template-curly-in-string */
```

Comment:
if this is causing issues in this file, it should be only on the line you want to use it for.